### PR TITLE
added enless wireless transmitters

### PR DESCRIPTION
**Pull request by reghund1986**
#### Summary

...

#### Changes

- ...
- ...

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/enless-wireless/600-021.png
Comment (LDannijs) [inline vendor/enless-wireless/600-021.png:1]: Could you please upload transparent images of the devices? And preferably without text around it, so just the device itself.

File: vendor/enless-wireless/tx-amb-600-022.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+name: TX VOC/T&H AMB 600-022
```
Comment (LDannijs) [inline vendor/enless-wireless/tx-amb-600-022.yaml:1]: Could you add a category to the name? So something along the lines of `TX VOC/T&H AMB 600-022 - Air Quality Sensor` (This is an example) 

Please do this for all devices as well.

File: vendor/enless-wireless/600-021.png
Comment (reghund1986) [reply vendor/enless-wireless/600-021.png:1]: Thank you , we will add the transparent image .

File: vendor/enless-wireless/tx-amb-600-022.yaml (hunk 1 above)
Comment (reghund1986) [reply vendor/enless-wireless/tx-amb-600-022.yaml:1]: Ok will add the category name

**Comment by Jaime-Trinidad** [comment]
@reghund1986 please sign the CLA

**Comment by LDannijs** [comment]
@reghund1986 could you sign the CLA? The PR can then be merged.

**Comment by LDannijs** [comment]
> [CLA assistant check](https://cla-assistant.io/TheThingsNetwork/lorawan-devices?pullRequest=721) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/TheThingsNetwork/lorawan-devices?pullRequest=721) before we can accept your contribution.**1** out of **2** committers have signed the CLA.✅ Jaime-Trinidad❌ reghund1986You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/TheThingsNetwork/lorawan-devices?pullRequest=721) it.

@reghund1986 since you are the one who created this PR you must sign the CLA yourself, click on the yellow button to sign it. Thank you.

**Comment by LDannijs** [comment]
@reghund1986 any update here? Can you still add transparent images for the devices?

**Pull request closed without merging**